test/functional/dd-school-data-spec.js

```diff
@@ -18,7 +18,8 @@ fdescribe( 'The dynamic financial aid disclosure', function() {
     expect( page.totalProgramDebt.getText() ).toEqual( '23,000' );
   } );
 
-  it( 'should dynamically display the completion rate if it\'s available', function() {
+  /* Note: this item was removed from the settlement version of the code */
+  /* it( 'should dynamically display the completion rate if it\'s available', function() {
```

Comment:
Ah, cool! The test will still be valid for the non-settlement path (which will be more clearly separated out from settlement in a task next sprint), so I'll follow your suggestion to use `xit`.